lms/static/sass/discussion/_discussion.scss

```diff
@@ -235,6 +235,10 @@ body.discussion {
     max-width: 800px;
     min-height: 500px;
     border-radius: 3px;
+
+    img {
```

Comment:
This will only work on the discussion board, and not for inline discussions or team discussions. :-( I wonder what the one true class is to put this on.